starter-code/src/components/FoodBox.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+export default class FoodBox extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  onChangeCount = (event) => {
+      this.setState({ count: event.target.value })
```

Comment:
Deberías tener una restricción para impedir añadir cualquier alimento cuya cantidad sea menor que la unidad (`if(event.target.value > 0)`). Además, de la manera en la que estás cambiando el estado te limitas a cambiar el contador pero no lo acumulas, evitando que puedas añadir alimentos de uno en uno o de dos en dos, por ejemplo. Para poder acumular dicha cantidad debería ser `this.setState({ count: this.state.count + event.target.value })`